runtime/near-vm-logic/src/types.rs

```diff
@@ -35,3 +37,5 @@ pub enum PromiseResult {
     Successful(Vec<u8>),
     Failed,
 }
+
+pub type ProfileData = Rc<RefCell<Vec<u64>>>;
```

Comment:
Could we try `Rc<RefCell<[u8; Actions::count() + ExtCosts::count() + 1]>>` once `count()` is `const`?